### PR TITLE
[tapd]fix some bug when first collect

### LINKED ISSUE #2373: [Bug][Tapd] tapd plugin cannot run without any data
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

tapd plugin cannot run without any data

### What you expected to happen

tapd collect data success.

### How to reproduce

just run it.

### Anything else

_No response_

### Version

main

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by likyh**
# Summary

fix some bug when first collect

### Does this close any open issues?
close #2373

### CODE REVIEW

File: plugins/tapd/tapd.go [hunk 1]
```diff
@@ -39,6 +39,10 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
+	err = cmd.MarkFlagRequired("company")
```
Comment (warren830) [inline plugins/tapd/tapd.go:42]: this is not required

**Review by warren830 — COMMENTED** [review]
only a minor modification needs to revert

File: plugins/tapd/tasks/shared.go [hunk 2]
```diff
@@ -115,19 +115,24 @@ type TapdApiParams struct {
...
+	} else {
+		filteredData.Options.CompanyId = 0
 	}
 	RawDataSubTaskArgs := &helper.RawDataSubTaskArgs{
 		Ctx:    taskCtx,
 		Params: params,
 		Table:  rawTable,
 	}
-	return RawDataSubTaskArgs, data
+	return RawDataSubTaskArgs, &filteredData
```
Comment (abeizn) [inline plugins/tapd/tasks/shared.go:137]: why not return 'data'?

Comment (likyh) [reply plugins/tapd/tasks/shared.go:137]: avoid using companyId but forget to set useCompanyId=true.

File: plugins/tapd/tapd.go (hunk 1 above)
Comment (likyh) [reply plugins/tapd/tapd.go]: fixed

**Review by abeizn — APPROVED** [review]
LGTM

**Pull request merged**